src/common/metrics/ngx_wa_histogram.c

```diff
@@ -222,6 +226,7 @@ ngx_wa_metrics_histogram_get(ngx_wa_metrics_t *metrics, ngx_wa_metric_t *m,
 
     for (i = 0; i < slots; i++) {
         h = m->slots[i].histogram;
+        out->sum += h->sum;
```

Comment:
Could this be `out->sum = h->sum` to be more explicit to the reader?

Reply:
`out->sum` is meant to accumulate the sum of each slot's histogram, hence the `+=`.
If we change value attribution to `=`, `out->sum` will no longer hold the accumulated sum but the histogram's sum of the last slot.

Reply:
Oh yes my bad, I misread this for the FFI function...